File: backend/src/main/java/com/victorvilar/projetoempresa/enums/VehicleType.java
package com.victorvilar.projetoempresa.enums;

public enum VehicleType {

    CAMINHAO_COMPACTADOR("Caminhão Compactador"),
    CAMINHAO_POLIGUINDASTE("Caminhão Poliguindaste"),
    CAMINHAO_ROLLON_ROLLOFF("Caminhão RollOn RollOff"),
    CAMINHAO_BAU("Caminhão Báu"),
    CAMINHAO_TANQUE("Caminhão Tanque"),
    FIORINO_FURGAO("Fiorino Furgão");

    private String type;

    VehicleType(String type){
        this.type=type;
    }

    public String getType(){
        return this.type;
    }


}

File: frontend/src/app/shared/enums/VehicleType.ts

export const VehicleType ={

  CAMINHAO_COMPACTADOR:"Caminhão Compactador",
  CAMINHAO_POLIGUINDASTE:"Caminhão Poliguindaste",
  CAMINHAO_ROLLON_ROLLOFF:"Caminhão Rollon Rolloff",
  CAMINHAO_BAU:"Caminhão Baú",
  CAMINHAO_TANQUE:"Caminhão Tanque",
  FIORINO_FURGAO:"Fiorino Furgão"

} as const;

export type VehicleTypeType = typeof VehicleType[keyof typeof VehicleType]

